index.js

```diff
@@ -62,7 +66,40 @@ async function executeQuery(query) {
   });
 }
 
-app.get("/", async (req, res) => {
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, 'views', '/index.html'));
+});
+
+app.post('/', async (req, res) => {
+  const queryText = req.body.userQuery;
+  const tables = await databaseMetadata();
+
+  const completion = await openai.createChatCompletion({
+    temperature: 0.1,
+    model: process.env.OPENAI_CHAT_MODEL,
+    messages: [
+      {
+        role: 'system',
+        content: `You are a helpful assistant that knows a lot about SQL language and manages a database.
+          You are using Postgres 12.
+          The database tables are: ${tables}
+          Please answer with each name between the char \` .
```

Comment:
Adding an example may be helpful to ensure its not answering with extra text or explanations, like:
```suggestion
          Please answer with each name between the char \` .
          Answer only with a comma separated list of tables, without any explanation. Example response: "`users`, `products`"
```